fixtures/varytec/giga-bar-frost-pix-8-rgb.json

```diff
@@ -0,0 +1,244 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Giga Bar Frost Pix 8 RGB",
+  "categories": ["Pixel Bar", "Color Changer"],
+  "meta": {
+    "authors": ["Luke Nelson"],
+    "createDate": "2022-10-12",
+    "lastModifyDate": "2022-10-12",
+    "importPlugin": {
+      "plugin": "qlcplus_4.12.1",
+      "date": "2022-10-12",
+      "comment": "created by Q Light Controller Plus (version 4.12.6)"
+    }
+  },
+  "links": {
+    "manual": [
+      "https://images.static-thomann.de/pics/atg/atgdata/document/manual/c_460683_460684_v2_en_online.pdf"
+    ],
+    "productPage": [
+      "https://www.thomann.de/gb/varytec_giga_bar_frost_pix_8_rgb.htm"
+    ],
+    "video": ["https://youtu.be/PixpS09Mc-I"]
+  },
+  "physical": {
+    "dimensions": [1020, 80, 65],
+    "weight": 1.5,
+    "power": 25,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [30, 30]
+    }
+  },
+  "matrix": {
+    "pixelCount": [8, 1, 1],
+    "pixelGroups": {
+      "Master": "all"
+    }
+  },
+  "templateChannels": {
+    "Red $pixelKey": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green $pixelKey": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue $pixelKey": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    }
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ShutterStrobe",
+        "shutterEffect": "Strobe",
+        "speedStart": "slow",
+        "speedEnd": "fast",
+        "helpWanted": "At which DMX values is strobe disabled?"
+      }
+    },
+    "Multi-function": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 7],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Closed",
+          "switchChannels": {
+            "Speed / Sensitivity": "No function"
+          }
+        },
+        {
+          "dmxRange": [8, 15],
+          "type": "ColorPreset",
+          "comment": "Constant uni-colored pattern in red",
+          "colors": ["#ff2600"],
+          "switchChannels": {
+            "Speed / Sensitivity": "No function"
+          }
+        },
+        {
+          "dmxRange": [16, 23],
+          "type": "ColorPreset",
+          "comment": "Constant uni-colored pattern in red and green",
```

Comment:
Is the pattern *uni-colored* or *red **and** green*? :wink: 
I guess that this is a low-resolution "yellow" but if the colors are not mixing well, we can indeed call this red+green.

Reply:
The manual is not great :') The matrix is split into segments, so from the language I imagine it's blocks of red and green. I can test the fixture at the weekend and report back.